site/docs/commands/test.md

```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguements
+
+The `flutter test` command exposes more arguements than those available through `very_good test`. Despite this, you can use the arguement terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options.
```

Comment:
argument is misspelled here too. should read "The flutter test command exposes more arguments than those available through very_good test. Despite this, you can use the argument terminator -- to signify the end of very_good test command options and the beginning of flutter test command options"